### PR TITLE
added isolate service example

### PR DESCRIPTION
**Pull request by DisDis**

### CODE REVIEW

**Comment by DisDis** [comment]
@sigurdm ping

**Comment by DisDis** [comment]
@sigurdm ping

**Comment by bordunosp** [comment]
ping

File: example/isolate-service/README.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+# Description
```
Comment (mraleph) [inline example/isolate-service/README.md:1]: This seems to be copied verbatim from the hello world folder and needs some updates, e.g. you can explain that this is an example of load balancing using multiple isolates listening to the same socket which is made possible by passing `shared: true` to `Server.serve`. 

I would also rename the folder from `isolate-service` to `helloworld-with-isolates`.

File: example/isolate-service/bin/server.dart [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+
+int _isolateId = 0;
+Future<Future<Object> Function()> runService(
+    Runner runner, dynamic address, int port) async {
+  final SendPort stopPort =
+      await runner.run(_startIsolateServer, [address, port, ++_isolateId]);
+  return () => _sendStop(stopPort);
+}
+
+Future _sendStop(SendPort stopPort) => singleResponseFuture(stopPort.send);
```
Comment (mraleph) [inline example/isolate-service/bin/server.dart:108]: It's unclear why this is a separate function. Can't it be folded into caller?

File: example/isolate-service/bin/server.dart [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+  final dynamic address = args[0];
+  final int port = args[1];
+  initLog(args.length > 2 ? "I${args[2]}" : null);
+
+  final server = Server([GreeterService()]);
+  await server.serve(address: address, port: port, shared: true);
+  _log.info('Service listening on port ${server.port}...');
+
+  return singleCallbackPort((SendPort resultPort) {
+    _log.info('Service shutdowning...');
```
Comment (mraleph) [inline example/isolate-service/bin/server.dart:120]: `Service shutting down`

**Review by mraleph — CHANGES_REQUESTED** [review]

**Comment by mit-mit** [comment]
@DisDis sorry for the (long) initial delay on review here. Are you still interested in working on this one?

**Comment by mraleph** [comment]
Closing as assumed stale for now.

**Pull request closed without merging**